app/assets/src/components/ui/containers/toast_container.scss

```diff
@@ -9,7 +9,7 @@
       margin-bottom: 5px;
       background: transparent;
 
-      &-body {
+      & .body {
```

Comment:
curious, did this change underneath?

Reply:
Sass started complaining about this after I updated it - `Invalid parent selector ":global(.Toastify__toast)"`.

Reply:
Wouldn't that be an error on the selector above? Was the error not specific?

Reply:
It said the above and specified this line (12). Perhaps I misread it

Reply:
Probably not if this fixed it! Just found it curious how the error gets reported.